packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.base.tsx

```diff
@@ -90,11 +94,12 @@ export class SwatchColorPickerBase extends BaseComponent<ISwatchColorPickerProps
     }
     return (
       <Grid
+        positionInSet={ positionInSet && positionInSet }
+        setSize={ setSize && setSize }
+        {...htmlProps}
         items={ colorCells.map((item, index) => { return { ...item, index }; }) }
         columnCount={ columnCount }
         onRenderItem={ this._renderOption }
-        positionInSet={ positionInSet && positionInSet }
-        setSize={ setSize && setSize }
```

Comment:
Why did these get move in front of htmlProps? If it's because those are getting return as part of the htmlProps, then you might want to consider including those attributes in the exclude list in the getNativeProps call

Reply:
Will do, I'll exclude it. I was thinking that we should no longer set these types manually and was trying to push to use the native supported for aria attributes.